packages/wonder-blocks-theming/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@khanacademy/wonder-blocks-theming",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
```

Comment:
**question:** Should this be a private package until it's ready for release?

Reply:
I think it's ok to have this be public - even if we release as an early 0.0.1 that's fine!

Reply:
FYI This won't be released yet as it will be initially landed in a separate branch.

My plan is to land all the theming-related PRs into this branch (`feature/theming`), but I can land all of this directly in `main` instead. Do y'all have a workflow preference?

Reply:
If I land everything in `feature/theming`, then I'll have to create another PR at the end (which includes all the changes) to land everything in `main`. We've used this approach in the past with `Cell` and `Banner`, but happy to use `main` directly if there's fit.